### ISSUE TITLE
Add h1 heading to news layout so consistent with other pages

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#829]
Page layout has h1 heading which is pulled from page's title field See [here](https://github.com/open-sdg/open-sdg/blob/master/_layouts/page.html). However, the news layout doesn't have a h1 heading.

Think we need to add same thing from page layout [here](https://github.com/open-sdg/open-sdg/blob/master/_layouts/news.html#L7), which will allow title to be pulled in from countries news page.

Not having a h1 heading is an accessibility issue as screenreaders rely on it to provide context.

Page 18 in DAC report

**Comment by LucyGwilliamAdmin** [comment]
Also, this is the ticket re. wider changes that @phillipgardiner mentioned:
https://github.com/ONSdigital/sdg-indicators/issues/3397
